doc/source/getting_started/faqs.rst

```diff
@@ -112,46 +113,56 @@ environment*:
 
 #. In a command window, use this code to set up and activate a local Python
    virtual environment::
-      
-    python -m venv pyenv   # Set up a local virtual environment
-    pyenv\Scripts\activate   # Activate the virtual environment on Windows
-    source pyenv/bin/activate.csh   # Activate the virtual environment on Linux (csh)
-    . pyenv/bin/activate   # Activate the virtual environment on Linux (bash)
 
+   .. code:: bash
+
+   .. vale off
```

Comment:
@jorgepiloto I had to deactivate vale for this code block because it was complaining about not knowing the words 'venv' and 'csh'.  Do you have any idea why I would have needed to deactivate it for a code block?  I would assume Vale is ignoring code blocks.

Reply:
I think the issue might be on the indentation of the code-snippet... it should be two spaces in. Hopefully that should solve the issue

Reply:
```rst
   .. code:: bash

      python -m venv venv   # Set up a local virtual environment
      venv\Scripts\activate   # Activate the virtual environment on Windows
      source venv/bin/activate.csh   # Activate the virtual environment on Linux (csh)
      . venv/bin/activate   # Activate the virtual environment on Linux (bash)

Reply:
Sorry, I got late to this ping. Glad to see everything got fixed.

Reply:
Thanks guys, let me try that.